### DISCUSSION TITLE
add experimental data as solution to speed up the training

### DISCUSSION THREAD

**Discussion by tjboise** [lululxvi/deepxde#1164] [Q&A]
Hi guys, in a forward problem, how can I add some experimental data as the solution to the PINN?

```
def func(x):
    return np.sin(k0 * x[:, 0:1]) * np.sin(k0 * x[:, 1:2])

data = dde.data.PDE(
    geom,
    pde,
    bc,
    num_domain=nx_train ** 2,
    num_boundary=4 * nx_train,
    solution=func, # I want to change the func to some experimental data, how could I do?
    num_test=nx_test ** 2,
)
```

**Comment by haison19952013** [comment]
Hi, plz refer to the section of inverse problem in the library document to see..https://deepxde.readthedocs.io/en/latest/demos/pinn_inverse.html

**Comment by divyanshjnv1** [comment]
@tjboise Did you get the way to add the experimental data points to speed up the pinn training , Please let me know, how to do that .
thanks!
